build.nix

```diff
@@ -220,6 +226,25 @@ let
       log "cargo_test_options: $cargo_test_options"
       log "cargo_bins_jq_filter: $cargo_bins_jq_filter"
       log "cargo_build_output_json (created): $cargo_build_output_json"
+      log "crate_sources: $crate_sources"
+      log "RUSTFLAGS: $RUSTFLAGS"
+      log "CARGO_BUILD_RUSTFLAGS: $CARGO_BUILD_RUSTFLAGS"
+
+      ${lib.optionalString remapPathPrefix ''
+
+      # Remove the source path(s) in Rust
```

Comment:
Maybe indent this block so that you can clearly see where the block begins and ends. But it's style so your call.

Reply:
let's ask `nixpkgs-fmt`

Reply:
`nixpkgs-fmt` is happy, though it does complain about `symlinkJoinPassViaFile` _cough cough_